clipboard.go

```diff
@@ -79,6 +79,8 @@ const (
 	FmtText Format = iota
 	// FmtImage indicates image/png clipboard format
 	FmtImage
+	// CF_DIB bitmap image. Windows only
+	FmtImageDib
```

Comment:
I like this contribution. But despite we documented this format is windows only, we still need to support other platforms, including macOS and Linux.

Reply:
Another thought: if Dib is essentially an image, why not just merge it into FmtImage. If DIBV5 is not recognizable on windows, then we do another try on the DIB format. If it works, we convert the DIB into PNG and return the data.

Reply:
> Another thought: if Dib is essentially an image, why not just merge it into FmtImage. If DIBV5 is not recognizable on windows, then we do another try on the DIB format. If it works, we convert the DIB into PNG and return the data.

Good idea. Updated.

Reply:
Thanks! Then this format declaration is not necessary anymore.

Reply:
Updated